src/file-3.xoo

```diff
@@ -0,0 +1,3 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:18746/organizations/org1/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:18746/project/issues?id=sample&pullRequest=4982&issues=AWHDHASQ3TFC5WF5jaEs&open=AWHDHASQ3TFC5WF5jaEs)

[ISSUE_KEY]: AWHDHASQ3TFC5WF5jaEs